test/unit/org/apache/cassandra/db/SchemaCQLHelperTest.java

```diff
@@ -479,4 +505,44 @@ public void testBooleanCompositeKey() throws Throwable
         execute("insert into %s (t_id, id, ck, nk) VALUES (true, true, false, true)");
         assertRows(execute("select t_id, id, ck, nk from %s"), row(true, false, false, true), row(true, true, false, true));
     }
+    
+    @Test
+    public void testParseCreateTableWithDroppedColumns()
+    {
+        String keyspace = createKeyspace("CREATE KEYSPACE %s WITH replication={ 'class' : 'SimpleStrategy', 'replication_factor' : 1 }");
+        String createTable = "CREATE TABLE IF NOT EXISTS %s (\n" +
+                             "    pk1 varint,\n" +
+                             "    ck1 varint,\n" +
+                             "    PRIMARY KEY (pk1, ck1)\n" +
+                             ") WITH ID = 552f4510-b8fd-11eb-aef4-518b3b328020\n" +
+                             "    AND CLUSTERING ORDER BY (ck1 ASC)\n" +
+                             "    AND DROPPED COLUMN RECORD reg1 varint USING TIMESTAMP 10000\n" +
+                             "    AND DROPPED COLUMN RECORD st1 varint static USING TIMESTAMP 5000\n";
+        createTable(keyspace, createTable);
+    }
+
+    @Test
+    public void testParseCreateTableWithDuplicateDroppedColumns()
+    {
+        String keyspace = createKeyspace("CREATE KEYSPACE %s WITH replication={ 'class' : 'SimpleStrategy', 'replication_factor' : 1 }");
+        String createTable = "CREATE TABLE IF NOT EXISTS %s (\n" +
+                             "    pk1 varint,\n" +
+                             "    ck1 varint,\n" +
+                             "    PRIMARY KEY (pk1, ck1)\n" +
+                             ") WITH ID = 552f4510-b8fd-11eb-aef4-518b3b328020\n" +
+                             "    AND CLUSTERING ORDER BY (ck1 ASC)\n" +
+                             "    AND DROPPED COLUMN RECORD reg1 varint USING TIMESTAMP 10000\n" +
```

Comment:
I assume it is not possible to get such situation (two records for the same column) in the reality.